### ISSUE TITLE
Pistols Only Status

### ISSUE THREAD

**Issue by KroniK907** [jaredballou/insurgency-sourcemod#19]
Hello,

What is the current status of the pistols only plugin? I was looking through the code and trying to figure out if it just wasn't finished, or if there was some bug preventing it from working properly.

Thanks

**Comment by jaredballou** [comment]
It's been deprecated in favor of theaters for now. There wasn't a good programmatic way to limit kits to pistols only. There is also the theater creation tool, which allows you to turn groups of items as removed, given to all players, or as the only items available.

**Comment by KroniK907** [comment]
That makes sense. In competitive matches, there is often a pistol round so I thought it might be useful for that when you want to override the theater for 1 round or something.

**Comment by jaredballou** [comment]
Good point. I was working on a more in-depth item management plugin, so that admins can add and remove items from players. If you want something simple, it might be workable as a base, but it's been so long since I worked on it that I can't confirm it works at all.

**Comment by KroniK907** [comment]
Speaking of competitive (and on an unrelated note) do you think it would be technically possible for me to create a plugin that can easily "pause" a game such that no points are gained, no objectives can be captured, the round time is extended, and if possible, no one can be killed, and then reverted back to normal once the pause is removed?

**Comment by jaredballou** [comment]
Set sv_timescale to 0 maybe? I'm not sure.
